src/elements/content-sidebar/__tests__/withSidebarAnnotations.test.js

```diff
@@ -0,0 +1,267 @@
+        `(
+            'should $title',
+            ({ isOpen, isPending, isAnnotationsPath, hasItems, expectedAddCount, expectedRefreshCount }) => {
+                const annotatorStateMock = {
+                    action: isPending ? 'create_start' : null,
+                    annotation: {},
+                    meta: {
+                        requestId: '123',
+                    },
+                };
+
+                const wrapper = getWrapper({ annotatorState: annotatorStateMock, currentUser, isOpen });
+                const instance = wrapper.instance();
+
+                annotatorContextProps.getAnnotationsMatchPath.mockReturnValueOnce(isAnnotationsPath);
+                feedAPI.getCachedItems.mockReturnValueOnce({ items: hasItems });
+                instance.sidebarPanels = {
+                    current: {
+                        refresh: jest.fn(),
+                    },
+                };
+
+                instance.addAnnotation();
+
+                expect(feedAPI.addAnnotation).toHaveBeenCalledTimes(expectedAddCount);
+                if (expectedAddCount) {
+                    expect(feedAPI.addAnnotation).toBeCalledWith(
+                        file,
+                        currentUser,
+                        annotatorStateMock.annotation,
+                        annotatorStateMock.meta.requestId,
+                        isPending,
+                    );
+                }
+                expect(instance.sidebarPanels.current.refresh).toHaveBeenCalledTimes(expectedRefreshCount);
+            },
+        );
+    });
+
+    describe('updateActiveAnnotation()', () => {
+        test.each`
+            activeAnnotationId | fileVersionId | location                                    | expectedPath                       | expectedState
+            ${'234'}           | ${'456'}      | ${{ pathname: '/' }}                        | ${'/activity/annotations/456/234'} | ${{ open: true, silent: false }}
+            ${'234'}           | ${undefined}  | ${{ pathname: '/' }}                        | ${'/activity/annotations/123/234'} | ${{ open: true, silent: false }}
+            ${null}            | ${'456'}      | ${{ pathname: '/' }}                        | ${'/activity/annotations/456'}     | ${undefined}
+            ${null}            | ${'456'}      | ${{ pathname: '/', state: { foo: 'bar' } }} | ${'/activity/annotations/456'}     | ${{ foo: 'bar' }}
+            ${'234'}           | ${'456'}      | ${{ pathname: '/', state: { foo: 'bar' } }} | ${'/activity/annotations/456/234'} | ${{ foo: 'bar', open: true, silent: false }}
+        `(
+            'should set location path based on match param fileVersionId=$fileVersionId and activeAnnotationId=$activeAnnotationId',
+            ({ activeAnnotationId, fileVersionId, location, expectedPath, expectedState }) => {
+                const annotatorState = {
```

Comment:
Should this variable live near the others in the test? It doesn't seem to be used until later.